src/tv2-common/uniformConfig.ts

```diff
@@ -15,6 +15,8 @@ export interface UniformConfig {
 		nextAux?: SwitcherAuxLLayer
 		/** Optional layer to show the jingles on an USK */
 		jingleUskMixEffect?: SwitcherMixEffectLLayer
+		/** Optional layer to show Fullscreen graphics on an USK */
```

Comment:
What does `USK` stand for? 
I'm sure it's a broadcast term that I don't know yet :)

Reply:
Wait, is it `Up Stream Key`?